label_sync/BUILD.bazel

```diff
@@ -34,7 +34,7 @@ docker_push(
 # bazel-bin/.../label_sync loads the image into docker
 # bazel-bin/.../label_sync.binary is the binary
 go_image(
-    name = "label_sync-image",
```

Comment:
There are other places where this target is used (L22-L24, aka the target to build and push this image). Please address. Same for commenter-image